src/components/Text/text/text.ts

```diff
@@ -0,0 +1,87 @@
+import {block} from '../../utils/cn';
+
+import './text.scss';
+
+const b = block('text');
+
+export const TYPOGRAPHY_VARIANTS = [
+    'display-4',
+    'display-3',
+    'display-2',
+    'display-1',
+    'header-2',
+    'header-1',
+    'subheader-3',
+    'subheader-2',
+    'subheader-1',
+    'body-3',
+    'body-2',
+    'body-1',
+    'body-short',
+    'caption-2',
+    'caption-1',
+    'code-3',
+    'code-inline-3',
+    'code-2',
+    'code-inline-2',
+    'code-1',
+    'code-inline-1',
+] as const;
+
+export interface TextBase {
+    /**
+     * Storybook: https://preview.yandexcloud.dev/uikit/?path=/story/typography--common
+     *
+     * - body:
+     *      - 1: font-size: 13px; line-height: 18px; (**Default variant**)
+     *      - 2: font-size: 15px; line-height: 20px;
+     *      - 3: font-size: 17px; line-height: 24px;
+     *      - short: font-size: 13px; line-height: 16px;
+     * - caption:
+     *      - 1: font-size: 9px; line-height: 12px;
+     *      - 2: font-size: 11px; line-height: 16px;
+     * - header:
+     *      - 1: font-size: 20px; line-height: 24px; font-weight: 900;
+     *      - 2: font-size: 24px; line-height: 28px; font-weight: 900;
+     * - subheader:
+     *      - 1: font-size: 13px; line-height: 18px; font-weight: 900;
+     *      - 2: font-size: 15px; line-height: 20px; font-weight: 900;
+     *      - 2: font-size: 17px; line-height: 24px; font-weight: 900;
+     * - display:
+     *      - 1: font-size: 28px; line-height: 36px; font-weight: 900;
+     *      - 2: font-size: 32px; line-height: 40px; font-weight: 900;
+     *      - 3: font-size: 40px; line-height: 48px; font-weight: 900;
+     *      - 4: font-size: 48px; line-height: 52px; font-weight: 900;
```

Comment:
`font-weight` is controlled by [variables](https://github.com/yandex-cloud/uikit/blob/main/styles/typography.scss#L9-L15), and can be changed outside.

Reply:
the main idea of this comments is ability to quick look at the component and understand what it does. Of course we can override every `token` of text variant.

I will add note about it in this comment